tcms/testcases/migrations/0006_merge_text_field_into_testcase_model.py

```diff
@@ -58,7 +62,7 @@ class Migration(migrations.Migration):
         ),
 
         # copy the data from the related model
-        migrations.RunPython(forward_copy_data),
+        migrations.RunPython(forward_copy_data, backward_empty_callable),
```

Comment:
Just found out `RunPython.noop` which doesn't do anything and is designed to be passed in migrations:
https://docs.djangoproject.com/en/3.0/ref/migration-operations/#django.db.migrations.operations.RunPython.noop

It's better to use it instead of creating a new empty function.